<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grading App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Final grade:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDidIPass">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Did I pass?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResult">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGrade">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
